src/handler.ts

```diff
@@ -56,6 +59,15 @@ export default class DataHandler {
    * @param event a browser Event or CustomEvent emitted
    */
   handleEvent(event: CustomEvent<DataLayerDetail>): void {
+    const handleEventSpan = Telemetry.startSpan(telemetryType.handleEventSpan, {
+      operatorCount: this.operators.length,
+      operatorNames: this.operators
+        .map((operator) => operator.options.name)
+        // Unique operator names
+        .filter((name, index, self) => self.indexOf(name) === index)
```

Comment:
Per my comment above, maintaining the order in which operators are executed (and all operators in the execution chain) could be insightful. Does this report all operators across all rules, rather than just the ones that apply to this particular event?

Reply:
Only rules applying to the event being processed (from the defined rule which triggered the event) would be included. I could see the argument for not de-duping like you mentioned and preserving original processing order.

Reply:
Ah, got it - I forgot that there's one `DataHandler` instance per rule. Yeah I think not de-duping would be better. The data is interesting and I don't think de-duping would save much in terms of payload size.